src/testjfxapp/Settings.java

```diff
@@ -37,11 +28,16 @@ public void start(Stage primaryStage) throws Exception {
         window = primaryStage;
 
         closeProgram = new Button("Quit");
-        closeProgram.setOnAction(e -> quitProgram());
-
+        closeProgram.setOnAction(e -> mainMenu.quitProgram());
 
         btm1 = new Button("Back to Menu");
-        btm1.setOnAction(e -> window.setScene(settings));
+        btm1.setOnAction(e -> {
+            try {
+                mainMenu.start(window);
+            } catch (Exception ex) {
+                ex.printStackTrace();
+            }
+        });
```

Comment:
We don't need to start the main menu, as it has already started, just need to set the scene back to the menu scene